Lib/tkinter/test/test_ttk/test_widgets.py

```diff
@@ -1485,6 +1485,17 @@ def test_insert_item(self):
             self.tv.insert('', 'end', text=value), text=None),
             value)
 
+        # test for values which are not None
+        itemid = self.tv.insert('', 'end', 0)
+        self.assertEqual(itemid, '0')
+        itemid = self.tv.insert('', 'end', 0.0)
+        self.assertEqual(itemid, '0.0')
+        # this is because False is resolveds to 0 and element with 0 id is already present
+        self.assertRaises(tkinter.TclError, self.tv.insert, '', '',
```

Comment:
No need to wrap this line, it is not long enough.

Why the second argument of `insert()` is `''` instead of `'end'`?

Reply:
My bad this should be end only.